db/seeds.rb

```diff
@@ -18,6 +18,13 @@ def format_errors(model = nil)
     ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
     User.current = nil
   end
+  if Setting['db_pending_seed']
```

Comment:
There are still situations, such as adding a plugin or upgrade that could reset this, but if we are ok with that then I'm fine with this.